plugwise_usb/connection/receiver.py

```diff
@@ -127,72 +129,63 @@ def connection_made(self, transport: SerialTransport) -> None:
             self._loop.create_task(
                 self._notify_stick_event_subscribers(StickEvent.CONNECTED)
             )
+        self._msg_processing_task = self._loop.create_task(
+                         self._msg_queue_processing_function()
+                     )
 
     async def close(self) -> None:
         """Close connection."""
+
         if self._transport is None:
             return
         if self._stick_future is not None and not self._stick_future.done():
             self._stick_future.cancel()
 
         self._transport.close()
+        self._msg_processing_task.cancel()
 
     def data_received(self, data: bytes) -> None:
         """Receive data from USB-Stick connection.
 
         This function is called by inherited asyncio.Protocol class
         """
-        _LOGGER.debug("USB stick received [%s]", data)
+        #_LOGGER.info("USB stick received [%s]", data)
```

Comment:
Uncomment code.
Lets log it either as debug or remove not at all.

Reply:
yes it's more debug^2 on the data layer, will remove them ... unless we want something to record dataflow for testing